$NetBSD: patch-src_perl_ps2eps,v 1.1 2025/05/14 06:52:11 wiz Exp $

Fix path to perl interpreter (further fixed by REPLACE_PERL).

--- src/perl/ps2eps.orig	2021-03-13 23:58:05.000000000 +0000
+++ src/perl/ps2eps
@@ -1,7 +1,4 @@
-eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}' && eval 'exec perl -S $0 $argv:q' # -*-perl-*-
-        if 0;
-# The expression in the previous line replaces the unix specific line 
-# {#!/usr/bin/perl}.   
+#!/usr/bin/perl
 # ps2eps - convert PostScript to EPS (Encapsulated PostScript) files
 # -------------------------------------------------------------------
 # (C)opyright 1998-2018 Roland Bless
